src/components/Form/CheckAclPermsDocContainerForm.jsx

```diff
@@ -93,30 +99,37 @@ const CheckAclPermsDocContainerForm = () => {
     }
   };
 
-  const formRowStyle = {
-    margin: '20px 0'
-  };
-
   return (
     <FormSection
       title="Check Permission to Documents Container"
       state={state}
       statusType="Permission status"
       defaultMessage="To be set..."
     >
-      <form onSubmit={handleAclPermission} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="set-acl-to">
-            Check permissions to username&apos;s Documents container:{' '}
-          </label>
-          <br />
-          <br />
-          <input id="set-acl-to" size="25" name="setAclTo" {...user} placeholder={selectedUser} />
-        </div>
-        <button disabled={state.processing} type="submit">
-          Check Permission
-        </button>
-      </form>
+      <Box display="flex" justifyContent="center" sx={{ minWidth: 120 }}>
+        <form onSubmit={handleAclPermission} autoComplete="off">
+          <FormControl fullWidth>
```

Comment:
just as a test I deleted the `fullWidth` here and stayed same size.

Reply:
This seems to be correct. The fullWidth attribute doesn't seem to affect the size of the form after testing it locally with the `fullWidths` removed with `<FormControl>`.